### PR TITLE
Moderation plugin PR

### PR DESCRIPTION
**Pull request by xPolar**

### CODE REVIEW

File: plugins/registry.json [hunk 1]
```diff
@@ -178,5 +178,14 @@
...
-    }
+    },
+    "moderation":{
+        "repository": "xPolar/modmail-plugins/Moderation/",
+        "branch": "master",
+        "description": "Moderate your server with ModMail, bring the Mod to Modmail!",
+        "bot_version": "3.0.3",
+        "title": "Moderate your server",
+        "icon_url": "https://cdn.discordapp.com/attachments/539943767562780704/601485194196680704/wGFmzZq.png",
+    }    "thumbnail_url": "https://cdn.discordapp.com/attachments/539943767562780704/601485194196680704/wGFmzZq.png"
```
Comment (Taaku18) [inline plugins/registry.json]: Syntax Error, need fixing

**Comment by Taaku18** [comment]
Btw, I see you set your required bot version to v3.0.3; if your plugin requires core features introduced in v3.0.3 disregard this comment. However, this will significantly reduce your userbase as many users are not always up-to-date.

**Comment by Taaku18** [comment]
Before I merge this, is the plugin thoroughly tested?

**Comment by xPolar** [comment]
Yes, the code is from my bot which has been heavily tested, so everything should be A ok. 👌

**Pull request merged**